miso-web/src/main/webapp/scripts/warning_poolorder.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+WarningTarget.poolorder = {
+  getWarnings: function(poolorder) {
+    return [{
+      include: poolorder.duplicateIndices,
+      tableMessage: Constants.warningMessages.duplicateIndices
+    }, {
+      include: poolorder.nearDuplicateIndices,
```

Comment:
Can a clause get added here: `&& (Constants.warningEditDistance > Constants.errorEditDistance)` ?
(When we get to the glorious only-automatic-analysis future, <=2 mismatches will be error and there will be no separate warning threshold; in that case, it should only display the duplicates warning)

Reply:
Makes sense. Will do.